docs/driver-only-builds.md

```diff
@@ -353,8 +353,11 @@ with the following conditions on the underlying key types:
   `MBEDTLS_CHACHA20_C` and algorithm `MBEDTLS_POLY1305_C`.
 
 It should be noticed that disabling `MBEDTLS_CIPHER_C` helps to reduce the
-code's footprint, but unfortunately it makes the following modules unavailable:
-- `MBEDTLS_PKCS[5|12]_C`,
-- `MBEDTLS_NIST_KW_C`.
+code's footprint, but unfortunately it makes the following features unavailable:
+- encryption/decryption in PKCS5 and PKCS12 modules (key derivations will still
+  be available),
+- encrypted PEM (write and unecrypted read work normally),
```

Comment:
I don't think that has anything to do with `CIPHER_C`, as `pem.c` does not use `mbedtls_cipher_xxx` functions.